### PR TITLE
Mana increased message

### PR DESCRIPTION
**Pull request by keepermanj**
A good reminder of how to use git. Simple code change. "You draw mana from the destruction of a tribe of bandits." etc. A better message because the old message assumed you knew about "The tribe...".

### CODE REVIEW

File: collective.cpp [hunk 1]
```diff
@@ -754,8 +754,9 @@ void Collective::onEvent(const GameEvent& event) {
         if (col->isDiscoverable())
           if (auto enemyId = col->getEnemyId()) {
             if (auto& name = col->getName())
-              control->addMessage(PlayerMessage("The tribe of " + name->full + " is destroyed.",
-                  MessagePriority::CRITICAL));
+              control->addMessage(PlayerMessage(
+                "You draw mana from the destruction of a tribe of " + name->full + ".",
+                MessagePriority::CRITICAL));
```
Comment (miki151) [inline collective.cpp]: Did you want to replace just the "destroyed" message or also the "surge of power"? Right now it seems you would get two messages about getting mana.

**Comment by SoftMonster** [comment]
Yeah. I thought it looked ok when I tested it. You do get both messages.

File: collective.cpp (hunk 1 above)
Comment (keepermanj) [reply collective.cpp]: This is OK, honest.

**Comment by SoftMonster** [comment]
Been trying to say that you do get both messages but that was expected. I can change it if you like.

**Comment by miki151** [comment]
I tested this and just prefer the old way, sorry.

**Comment by SoftMonster** [comment]
Ok

**Comment by SoftMonster** [comment]
We can close it if you like.

**Pull request closed without merging**